document/PolicyComplianceProfile.mdk

```diff
@@ -1 +1,65 @@
-# Policy Compliance Profile { #policy-compliance-profile }
\ No newline at end of file
+# Policy Compliance Profile { #policy-compliance-profile }
+
+## General { #policy-compliance-profile-general }
+
+It is sometimes desirable to use the output of a static analysis tool to help determine
+whether a project is in compliance with applicable policies.
+For example, a policy might state that "the code shall contain no null-dereference errors."
+To determine whether their code base complies with this policy,
+a team might run a tool that includes a rule that checks for null-dereference errors,
+and then examine the run log.
+The fact that the run log contains no results that violate this policy is not
+sufficient to determine compliance.
+The team must also be able to verify that the tool actually ran the necessary rule.
+
+The Policy Compliance Profile defines additional optional elements of the SARIF format
+that are intended for use in this scenario.
+
+## `runInfo` object { #policy-compliance-profile-runInfo }
+
+The Policy Compliance Profile defines one additional property on the `runInfo` object (&sect;[#runInfo]).
+
+### `complianceInfo` property { #runInfo-complianceInfo }
+
+A `runInfo` object (&sect;[#runInfo]) may contain a property named `complianceInfo`
+whose value is a `complianceInfo` object (&sect;[#complianceInfo]).
+
+## `complianceInfo` object { #complianceInfo }
+
+### General { #complianceInfo-general }
+
+A `complianceInfo` object contains information that helps determine whether the
+tool producing the log was configured in a way that complies with applicable policies.
+
+A `complianceInfo` object is not a policy or statement of policy.
+It is merely a collection of information that helps determine policy compliance.
+
+### `executedRules` property { #complianceInfo-executedRules }
+
+A `complianceInfo` object (&sect;[#complianceInfo])
+shall contain a property named `executedRules` whose value is an array
+of zero or more `executedRule` objects (&sect;[#executedRule]).
+
+## `executedRule` object { #executedRule }
```

Comment:
You might want to policheck the term 'executedRule'. IIRC, the term executed OK because the target audience is developers, but I think the preferred term is 'run' ... or something like that.